subsys/bluetooth/controller/hci_driver.c

```diff
@@ -273,6 +273,14 @@ static bool event_packet_is_discardable(const uint8_t *hci_buf)
 		switch (me->subevent) {
 		case BT_HCI_EVT_LE_ADVERTISING_REPORT:
 			return true;
+#if defined(CONFIG_BT_EXT_ADV)
+		case BT_HCI_EVT_LE_EXT_ADVERTISING_REPORT:
+		{
+			struct bt_hci_evt_le_ext_advertising_report *ext_adv = (void *)&hci_buf[3];
+			bool discardable = (ext_adv->adv_info->evt_type & BT_HCI_LE_ADV_EVT_TYPE_LEGACY);
+			return discardable;
```

Comment:
A `bt_hci_evt_le_ext_advertising_report` may contain several reports, right? 

Looks like this will only get the first report. May different reports in the event have different event types?

Reply:
you are right, this event type does support that. but the SDC only ever givs exteneded adv reports with num_reports==1. so we decided this code was okay to add on this release branch.